reference/5.1/Microsoft.PowerShell.Core/About/about_Jobs.md

```diff
@@ -15,34 +15,47 @@ expression in the background without interacting with the current session.
-|`RemoteJob`    |Command and script run on a remote computer.                 |
-|`BackgroundJob`|Command and script run in a separate process on the local    |
-|               |machine.                                                     |
-|`ThreadJob`    |Command and script run in a separate thread within the same  |
-|               |process on the local machine.                                |
-
-Each type of job has benefits and drawbacks. Running script remotely on a
-separate machine or in a separate process has great isolation. Any errors won't
-affect other running jobs or the client that started the job. But the remoting
-layer adds overhead, including object serialization. All objects passed to and
-from the remote session must be serialized and then deserialized as it passes
-between the client and the target session. The serialization operation can use
-many compute and memory resources for large complex data objects.
-
-This topic explains how to run background jobs in PowerShell on a local
-computer. For information about running background jobs on remote computers,
-see [about_Remote_Jobs](about_Remote_Jobs.md). For more information about
-thread jobs, see
-[about_Thread_Jobs](/powershell/module/microsoft.powershell.core/about/about_Thread_Jobs).
-
-When you start a background job, the command prompt returns immediately, even
-if the job takes an extended time to complete. You can continue to work in the
-session without interruption while the job runs.
+PowerShell concurrently runs commands and scripts through jobs. There are three
+jobs types provided by PowerShell to support concurrency.
+
+- `RemoteJob` - Commands and scripts run on a remote session. For information,
+  see [about_Remote_Jobs](about_Remote_Jobs.md).
+- `BackgroundJob` - Commands and scripts run in a separate process on the local
+  machine.
+- `PSTaskJob` or `ThreadJob` - Commands and script runs in a separate thread
+  within the same process on the local machine. For more information, see
+  [about_Thread_Jobs](/powershell/module/ThreadJob/about_Thread_Jobs).
+
+Running scripts remotely, on a separate machine or in a separate process,
+provide great isolation. Any errors that occur in the remote job do not affect
+other running jobs or the parent session that started the job. But the remoting
+layer adds overhead, including object serialization. All objects are serialized
+and deserialized as they are passed between the parent session and the remote
+(job) session. Serialization of large complex data objects can consume large
+amounts of compute and memory resources.
+
+Thread based jobs are not as robust as Remote and Background jobs, because they
+run in the same process on different threads. If one job has a critical error
+that crashes the process, then all other jobs in the process are terminated.
+
+However, thread-based jobs require less overhead. They don't use the remoting
```

Comment:
thread-based is hyphenated here, but not on line 37.  Should be consistent.

Reply:
fixed